elastic/integrations
# [ThreatConnect] Functionality updates.

- Enhancement

Remove default SSL certificate, update descriptions, add ability to control attributes, associated groups and indicators.

See title. 

## Checklist


## Author's Checklist

- [ ]  Confirm new bool inputs exist and default to FALSE

## How to test this PR locally

Install integration and confirm new fields exist, SSL field is empty, and descriptions and associated text is updated. 

## Related issues

- 

## Screenshots
